src/nvim/decoration.c

```diff
@@ -97,6 +114,7 @@ void decor_free(Decoration *decor)
       clear_virttext(&kv_A(decor->virt_lines, i).line);
     }
     kv_destroy(decor->virt_lines);
+    xfree(decor->sign_text);
```

Comment:
@bfredl do I need a NULL check here like I needed for extmark.c?

Reply:
`xfree` never needs a NULL check of its argument.

Reply:
Ok I must have got confused somewhere.